requirements.txt

```diff
@@ -1,2 +1,3 @@
-requests>=2.13.0
-google_trans_new
\ No newline at end of file
+git+https://github.com/NeonGeckoCom/neon_solver_wolfram_alpha_plugin
+quebra_frases
+neon-lang-plugin-libretranslate
```

Comment:
once the solvers are packaged this will be updated, this should be set where translation is used (neon_solvers main package)

this is just a workaround so PR can be tested with git installs of the solvers

quebra_frases also comes from neon_solvers